cassandra-migration/src/main/resources/db/migration/cassandra/V1.54_cassandra.cql

```diff
@@ -1 +1 @@
-insert into sunbird.role_group (id,name) values ('REPORT_ADMIN','Report Admin');
\ No newline at end of file
+insert into sunbird.role (id,name,rolegroupid,status) values ('REPORT_ADMIN’,’Report Admin’,['COURSE_MENTOR'],1);
```

Comment:
Change COURSE_MENTOR to REPORT_ADMIN.

Entries are required in both role and role_group tables.